src/button/internal.tsx

```diff
@@ -31,7 +31,7 @@ export type InternalButtonProps = Omit<ButtonProps, 'variant'> & {
     | (React.HTMLAttributes<HTMLAnchorElement> & React.HTMLAttributes<HTMLButtonElement>)
     | Record<`data-${string}`, string>;
   __iconClass?: string;
-  __activated?: boolean;
```

Comment:
looks like it's not being used anywhere (it was introduced for the TopNav a while ago, but that code has gone)